### PR TITLE
BXMSDOc-8456-main: Add optional step about Decision Manager only configuration.

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 1]
```diff
@@ -54,7 +54,17 @@ To use a Git hooks directory, make the following changes:
...
+. Optional: In the *SSH secret* field enter the name of the secret with the SSH key and known hosts files.
+. Optional: To configure {KIE_SERVER} for decision management only capabilities so that jBPM and case management features are disabled, complete either one of the following steps:
++
+* Select *Execute Kie Server only with Decisions capabilities*.
++
+* Set `KIE_SERVER_DECISIONS_ONLY` to `true` in the `image.yaml` file. For more information, see https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/main/kieserver/image.yaml#L263)[`image.yaml`] on GitHub.
++
+[NOTE]
+====
+`KIE_SERVER_DECISIONS_ONLY` is set to `true` by default for Red Hat Decision Manager templates. For more information, see https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/main/templates/decision/rhdm713-authoring-ha.yaml#L61).[`rhdm713-authoring-ha.yaml`] on GitHub.
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:66]: this doc is about operator, shouldn't this be using the operator property instead and use this env on the template's guide??

**Review by kaldesai — APPROVED** [review]
@csherrar Looks good to me!

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 2]
```diff
@@ -54,7 +54,17 @@ To use a Git hooks directory, make the following changes:
 +
 .. Under *GitHooks*, in the *Mount path* field, enter a fully qualified path for the directory, for example, `/opt/kie/data/git/hooks`.
 .. In the fields under *GitHooks Configuration object*, select the *Kind* of the object that provides the file (`ConfigMap`, `Secret`, or `PersistentVolumeClaim`) and enter the *Name* of the object. This object is automatically mounted on the {CENTRAL} pods in the path that you specified for the Git hooks directory.
-.. Optional: In the *SSH secret* field enter the name of the secret with the SSH key and known hosts files.
+. Optional: In the *SSH secret* field enter the name of the secret with the SSH key and known hosts files.
+. Optional: To configure {KIE_SERVER} for decision management only capabilities so that jBPM and case management features are disabled, complete either one of the following steps:
++
+* Select *Execute Kie Server only with Decisions capabilities*.
++
+* Set `KIE_SERVER_DECISIONS_ONLY` to `true` in the `image.yaml` file. For more information, see https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/main/kieserver/image.yaml#L263)[`image.yaml`] on GitHub.
```
[the remark this answers is not in the document]
Comment (spolti) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:62]: the same for the changes below

[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:62]: Ok to remove L62?

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:66]: Ok to remove the Note?

**Pull request merged**